build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -108,12 +108,99 @@ service Execution {
     option (google.api.http) = { post: "/v2/{instance_name=**}/actions:execute" body: "*" };
   }
 
+  // Execute an action remotely, allowing dynamic retrieval of directory and
+  // file inputs from the client where the client did not upload the inputs
+  // beforehand. The client must remain connected to the worker during the
+  // duration of the action execution.
+  //
+  // This call may not be implemented on all platforms.
+  //
+  // In order to execute an action, the client must first upload the best
+  // guesses for the `Action`'s file inputs, the
```

Comment:
The best guesses for the `Action`'s file inputs are not a must, right?

Reply:
Correct, clarififed with significant text updates in this section.

---
In reply to: [458985393](https://github.com/erikma/remote-apis/pull/1#discussion_r458985393) [](ancestors = 458985393)